src/test/java/org/commcare/fixtures/test/StorageBackedFixtureTest.java

```diff
@@ -0,0 +1,37 @@
+package org.commcare.fixtures.test;
+
+import org.commcare.core.parse.ParseUtils;
+import org.commcare.test.utilities.CaseTestUtils;
+import org.commcare.util.mocks.MockDataUtils;
+import org.commcare.util.mocks.MockUserDataSandbox;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.xml.util.InvalidStructureException;
+import org.javarosa.xml.util.UnfullfilledRequirementsException;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+import org.junit.Before;
+import org.junit.Test;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.IOException;
+
+/**
+ * @author Phillip Mates (pmates@dimagi.com)
+ */
+public class StorageBackedFixtureTest {
+    private MockUserDataSandbox sandbox;
+
+    @Before
+    public void setUp() {
+        sandbox = MockDataUtils.getStaticStorage();
+    }
+
+    @Test
+    public void queryIndexedLookup() throws XPathSyntaxException, UnfullfilledRequirementsException,
+            XmlPullParserException, IOException, InvalidStructureException {
+        ParseUtils.parseIntoSandbox(this.getClass().getResourceAsStream("/indexed-fixture.xml"), sandbox);
+
+        EvaluationContext ec =
```

Comment:
It might be worth some manual test in here (somehow!) to ensure that this isn't using the old fixture format.